js/renderer.js

```diff
@@ -56,8 +57,11 @@ class Renderer {
     }
 
     drawPlayer() {
-        if (this.level.player.grapple) { this.drawGrappleWire() }
+        if (this.level.player.grapple) {
+            this.drawGrappleWire();
+        }
         this.renderSprite(this.level.player);
```

Comment:
you added player to level.actors, so this is actually double rendering the player sprite